Makefile

```diff
@@ -264,9 +264,13 @@ deploy-api:
 	aws lambda update-function-code --function-name ien-$(ENV_NAME)-api --zip-file fileb://./terraform/build/api.zip --region $(AWS_REGION)
 	aws lambda update-function-code --function-name ien-$(ENV_NAME)-syncdata --zip-file fileb://./terraform/build/api.zip --region $(AWS_REGION)
 
-deploy-all: sync-app deploy-api
+deploy-all: sync-app backup-db deploy-api
 	@echo "Deploying Webapp and API"
 
+backup-db:
```

Comment:
This would create backups on dev and test too for each deploy. Which is totally fine. 

How do we limit / purge the number of backups that can be maintained?

Reply:
@vyasworks AWS limits the number of snapshots for a region to 100. 

So you think we should use all that limit to prod?

Reply:
Nope, just wondering if we need to purge or not. 

if we keep 2-3 latest backups that should be fine